src/wp-includes/user.php

```diff
@@ -2938,6 +2938,17 @@ function retrieve_password( $user_login = null ) {
 		return $errors;
 	}
 
+	/**
+	 * Filter whether to disable the retrieve password email.
+	 *
+	 * @since 6.0.0
+	 *
+	 * @param bool $disable True to disable. Default: false
+	*/
+	if ( apply_filters( 'disable_retrieve_password_notification_email', false ) ) {
+		return true;
+	}
```

Comment:
This is looking good, but the naming should be adjusted to avoid the double negative. The most recent filters for disabling emails take the form of `send_<name>_email`, ie. `send_site_admin_email_change_email` and `send_password_change_email`.

A good name might be `send_retrieve_password_email`. Then the logic will need to be inverted with a default value of `true`.

Reply:
Changed it! Have a look on it and let me know if there is anything else.